subprojects/packagefiles/sdl2/meson.build

```diff
@@ -1030,13 +1030,16 @@ if platform_is_winrt or platform_is_haiku
     all_sources += [cxx_sources]
 endif
 
+do_install = get_option('default_library') != 'static' or not meson.is_subproject()
```

Comment:
Shouldn't the not meson.is_subproject() be removed in favor of meson install --skip-subprojects?

Reply:
--skip-subprojects is badly behaved if `default_library=shared`, since then you must install libsdl2.so but it doesn't get installed.

This PR correctly installs the library if either of the following conditions are true:
- sdl2 is being built standalone, so you care about installing it as its own unique thing
- the default_library is such that you need it installed at runtime

Reply:
this was taken from eli's comment [here](https://github.com/mesonbuild/wrapdb/pull/323#issuecomment-1066197960) so it is best you discuss that with him

Reply:
I think it's wrong to make such special case here. Why is SDL special compared to all other projects? The real fix is install --tags runtime. You could want to install static libraries to be consumed by another project via pkg+config.

Reply:
SDL isn't special, let's update *all* wraps to do like this one.

Thanks for the suggestion.

Reply:
That makes absolutely no sense. If you really think that's the right thing to do, should be done in meson by default. But it's wrong.

Reply:
I may be wrong, but at least I'm consistently wrong, because I am indeed on record as saying it should be done in Meson by default.

It's an old discussion at this point, frankly. The relevant keywords are "gst-build" and "metaproject".

Reply:
For the record, I agree with @xclaesse. This makes no sense to me. I understand you are the maintainer, but shouldn't we be pushing Meson best practices in the WrapDB and not what our personal ideas are?